Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/DriverLicense.hs

```diff
@@ -124,6 +124,8 @@ verifyDL isDashboard mbMerchant (personId, _, merchantOpCityId) req@DriverDLReq
     Just driverLicense -> do
       unless (driverLicense.driverId == personId) $ throwImageError imageId1 DLAlreadyLinked
       unless (driverLicense.licenseExpiry > now) $ throwImageError imageId1 DLAlreadyUpdated
+      let rules = listToMaybe driverLicense.failedRules
```

Comment:
Just add one more check if `driverLicense` is `INVALID` then throw error contact support. You driver license class of vehicles are not supported. If it is `VALID` then don't do anything and say `DLAlreadyUpdated`